packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -316,9 +320,13 @@ module.exports = class Checkout extends Page {
      *
      * @param {String} Field name
      * @param {String} Value to set
+     * @param {Boolean} Whether to tab out of the field or not
      */
-    setFieldValue (fieldName, value) {
+    setFieldValue (fieldName, value, tabOut = false) {
         this.fields[fieldName].input.setValue(value);
+        if (tabOut) {
+            this.fields[fieldName].input.setValue(['Tab']);
```

Comment:
I'm a bit torn about whether this should live inside this function or be its own.
Also, small nit: you could assign `this.fields[fieldName].input` to a variable.

Reply:
Actually, I've had another idea, how about replacing the `tabOut` parameter with an `options` parameter, which could contain `tabOut`? Then you would call it like this, which I think is easier to read and makes extending it slightly easier

`checkout.setFieldValue('emailAddress', email, { tabOutOfField: true });`